test/python/circuit/classical/test_expr_constructors.py

```diff
@@ -14,67 +14,13 @@
 
 import ddt
 
-from qiskit.circuit import Clbit, ClassicalRegister, Instruction
+from qiskit.circuit import Clbit, ClassicalRegister
 from qiskit.circuit.classical import expr, types
 from test import QiskitTestCase  # pylint: disable=wrong-import-order
 
 
 @ddt.ddt
 class TestExprConstructors(QiskitTestCase):
-    def test_lift_legacy_condition(self):
-        cr = ClassicalRegister(3, "c")
-        clbit = Clbit()
```

Comment:
All of these tests ought to stay - they're still valid and the function still exists.  `IfElseOp` and `WhileLoopOp` still allows their conditions to be in the legacy format, so the tests could swap to those, which is spiritually what the function is now for (or just write the conditions without an instruction).

Reply:
Fixed in: https://github.com/Qiskit/qiskit/pull/13506/commits/9ad7e9eae0dd8aeacab4dffe22344f696569748c